lib/rest_in_peace.rb

```diff
@@ -47,10 +47,8 @@ def clear_changes
   end
 
   def update_attributes(attributes)
-    attributes.each do |key, value|
-      next unless respond_to?("#{key}=")
-      send("#{key}=", value)
-    end
+    warn 'DEPRECATION WARNING: the method update_attributes is deprecated, use method update.'
```

Comment:
Yeah, but that was under the assumption that we can replace it. We can't, that's why we moved it to `active_model_api.rb`. The existing behaviour should be left as is.